nnvm/include/nnvm/top/tensor.h

```diff
@@ -259,6 +259,17 @@ struct ClipParam : public dmlc::Parameter<ClipParam> {
   }
 };
 
+struct CropLikeParam : public dmlc::Parameter<CropLikeParam> {
+  Tuple<int> offsets;
+  bool center_crop;
```

Comment:
Handle the layout as well.

Reply:
The reason I didn't add layout as parameter is this operator's behavior is unrelated to layout. It just crops on the second and third axis.